operations/_scripts/generate/generate_buckets_identifiers.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -e
+
+GITHUB_IDENTIFIER="$(echo $($GITHUB_ACTION_PATH/operations/_scripts/generate/generate_identifier.sh) | tr '[:upper:]' '[:lower:]')"
+
+function generateIdentifiers () {
```

Comment:
Same as for the other function, naming and description would be nice to haves.

And quickly looking at it, I didn't understand what kind of Identifiers the function generates.
`generate_bucket_names()` or `generate_bucket_identifiers` perhaps would be more clear?